packages/caravan-psbt/package.json

```diff
@@ -61,6 +61,7 @@
     "jest": "^29.4.1",
     "jsdom": "24.0.0",
     "jsdom-global": "3.0.2",
+    "lodash": "^4.17.21",
```

Comment:
I prefer to avoid the burden of lodash, but I suppose it's okay since it's just a dev dependency. It' might be preferable to define a `cloneDeep` function instead of bringing in this lib.

Reply:
Yeah, I wrestled with the same and tried to use alternatives. Apparently there's a native option in JS now: [structuredClone](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone). Unfortunately Jest doesn't support it.